resolverFactory.js

```diff
@@ -4,14 +4,12 @@ import { matchPath } from './matchPath.js'
 
 export const resolverFactory = (resolveTs) => {
     return function (specifier, ctx, defaultResolve) {
-        let match
+        let match = matchPath && matchPath(specifier)
```

Comment:
`const` should be used here rather than `let`.

Reply:
Actually not, because it could change line 12